### PR TITLE
docs(pathwayelements): document query

### PR DESCRIPTION
**Pull request by nckhell**
Screenshot 2022-06-15 at 17 12 52
Screenshot 2022-06-15 at 17 13 02
Screenshot 2022-06-15 at 17 13 07

### CODE REVIEW

File: content/api-reference/queries/get-pathway-elements.mdx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+### Query
+
+```graphql
+query GetPathwayElements($pathway_id: String!) {
+  pathwayElements(pathway_id: $pathway_id) {
+    elements {
+      id
+      parent_id
+      name
+      status
```
Comment (rahulkeerthi) [inline content/api-reference/queries/get-pathway-elements.mdx:29]:
```suggestion
      status // one of 'active', 'done' or 'scheduled'
```

File: content/api-reference/queries/get-pathway-elements.mdx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      type
```
Comment (rahulkeerthi) [inline content/api-reference/queries/get-pathway-elements.mdx:30]:
```suggestion
      type // one of 'pathway', 'track', 'step', 'action' or 'trigger'
```

File: content/api-reference/queries/get-pathway-elements.mdx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      activity_type
+      start_date
+      end_date
```
Comment (rahulkeerthi) [inline content/api-reference/queries/get-pathway-elements.mdx:33]:
```suggestion
      end_date // for a completed element, this is the completion date of the final action element in it
```

**Comment by rahulkeerthi** [comment]
@nckhell some general suggestions, nothing strictly required except this one: https://github.com/nckhell/awell-developers/pull/31/files/3bd2d792d6e612290038e8718bdc377aa2374939#r898168429

File: content/api-reference/queries/get-pathway-elements.mdx [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+### Variables
+
+```json
+{
+  "pathway_id": "{{PATIENT_PATHWAY_ID}}"
+}
+```
+
+## Timeline view
+
```
[the remark this answers is not in the document]
Comment (nckhell) [reply content/api-reference/queries/get-pathway-elements.mdx:65]: Added!

File: content/api-reference/queries/get-pathway-elements.mdx [hunk 5]
```diff
@@ -0,0 +1,78 @@
+---
+title: Get pathway elements
+description: Query all the elements or building blocks of a pathway
+---
+
+Where pathway activities represent every granular activity related to the operation or orchestration of a pathway. The pathway elements query allows you to query the main elements or building blocks of a patient's pathway and their status. It's essentially a different lens to look at a pathway and can be useful to create specific types of visualizations like a timeline view.
```
[the remark this answers is not in the document]
Comment (nckhell) [reply content/api-reference/queries/get-pathway-elements.mdx]: Added!

**Comment by nckhell** [comment]
@rahulkeerthi Thanks, great comments and input. Thank you!

**Pull request merged**